src/components/Tag/Tag.scss

```diff
@@ -2,37 +2,69 @@
     display: flex;
     align-items: center;
 
-    padding: var(--space-50) var(--space-75);
-    border-radius: var(--border-radius);
+    padding: var(--space-25) var(--space-50);
+    gap: 4px;
+    border-radius: 4px;
```

Comment:
please use space variables instead of fixed values (check other places in the file)

Reply:
mehhhh but the standard radius is 3px :( But i'll go for var(--space-50) then. It just feels less logical than a short '4px'...